### ISSUE TITLE
[1.14.x] EntityViewRenderEvent.CameraSetup not fired

### ISSUE THREAD

**Issue by Snownee** [MinecraftForge/MinecraftForge#5911]
**Minecraft Version:** 1.14.3

**Forge Version:** 27.0.24

EntityViewRenderEvent.CameraSetup seems never fired

Register:

``` java
MinecraftForge.EVENT_BUS.register(Handler.class);
```

Method:

``` java
@SubscribeEvent
public static void cameraSetup(CameraSetup event)
{
    System.out.println("Event is fired");
}
```

**Comment by diesieben07** [comment]
The whole rendering code seems to have been moved around a lot. From what I can tell the closest to this is now in `ActiveRenderInfo#func_216772_a`. Can someone who understands more about rendering confirm or provide more insight?

**Comment by LexManos** [comment]
Since @tterrag1098 hasn't responded to this, and this hasn't been explained more then 'I want the old function' I'm going to unassign this to allow stalebot to close it if nobody wants to look into it.
We need to know why you're needing this event so we can best decide what new hook spot to add.

**Comment by Snownee** [comment]
I am trying to port this simple feature. Not sure if there is any other way to implement it.

![nimble](https://user-images.githubusercontent.com/16059084/62405564-8115e980-b5d1-11e9-977b-41a0d7649a55.gif)

**Comment by Mirsario** [comment]
I'm also making a mod that has some camera improvements in it, and I'd really like to have this event back and working.

![mc-camera](https://user-images.githubusercontent.com/6795251/62705129-81f5b380-b9f5-11e9-87ad-d75e095273a2.gif)

I've managed to get camera rotations semi-working for this **mockup** by calling glRotatef for GL_MODELVIEW matrix in EntityViewRenderEvent.FOVModifier event, but that's obviously dumb and not enough. And there's plenty of issues with my implementation, one of which is a huge amount of broken rotations in OptiFine (shadows and skies don't get rotated with the world), which probably says that the CameraSetup event is really needed, as OptiFine may need to call it somewhere for proper compatibility (That's if it resets the matrix somewhere, I have no idea if it does or not).
It also seems to me like 1.14 MC lacks any camera roll fields, which previous versions and this event had.

(I've got just a few hours of knowledge of Java & MC's internals, might be saying bs).
